.eslintrc.js

```diff
@@ -18,5 +18,11 @@ module.exports = {
     'react',
   ],
   rules: {
+    'max-len': 'off', // Kind of useless, for code style
+    'radix': 'off', // Useless, we wont be dealing with hex values
+    'global-require': 'off', // this one is fine because were requiring image filepaths,  
+    'react/forbid-prop-types': 'off', // this ideally would be fixed, but were in too deep 
+    'react/prop-types': 'off', // this ideally would be fixed, but were in too deep 
+    'no-unused-expressions': ["error", { "allowTernary": true }] // Although ternary isn't optimized, it is still useful
```

Comment:
I think we should leave this on. I took a look at all of the places where this rule gets violated and those should all be changed to if/else statements.

Reply:
Yeah, good call. Should be fixed now